scripts/corelli/calibration/database.py

```diff
@@ -7,118 +7,198 @@
 
 # File that defines functionality for Corelli calibration database
 
-import pathlib
 from datetime import datetime
+import enum
+import pathlib
 import re
-from typing import Optional
+from typing import List, Optional, Tuple, Union
 
-from mantid.dataobjects import TableWorkspace
-from mantid.api import WorkspaceGroup, Workspace
+from mantid.dataobjects import EventWorkspace, TableWorkspace,  Workspace2D
```

Comment:
Are we using `EventWorkspace` at all?

Reply:
I did it in case as a safety measure, in case instrument scientists passes it as input to `save_calibration_set` in their scripts. It's only used to retrieve the day when the wire-scan experiment was run. This day sets the day-stamp for the banks to be calibrated. Ideally the instrument scientist would be using the `Workspace2D` resulting from using `load_banks`, but I don't trust this will happen in 100% of the times.
If you want, I can remove it. Let me know

Reply:
It's up to you, I just noticed since it's been mostly `TableWorkspace` we've been dealing with. My rule of thumb is that if it's something useful and easy to maintain so be it. If it's a nasty corner case I don't bother to expose it to the user. Fine with me, your call.